includes/Plugin.php

```diff
@@ -225,6 +225,10 @@ public function register() {
 		$this->analytics = new Analytics();
 		add_action( 'init', [ $this->analytics, 'init' ] );
 
+		// TODO, make this is a property. Not property as issue with phpmd. 
```

Comment:
There's now `@SuppressWarnings(PHPMD.TooManyFields)` in this class, so there should be no warnings anymore.